app/controllers/evaluators_controller.rb

```diff
@@ -13,19 +13,20 @@ def index
   end
 
   def create
+    @evaluator_invitation = EvaluatorInvitation.new(evaluator_invitation_params)
+
     result = evaluator_service.process_evaluator_invitation(
       evaluator_invitation_params[:email],
       evaluator_invitation_params
     )
 
     if result[:success]
       redirect_to phase_evaluators_path(@phase), notice: result[:message]
-    else
-      flash.now[:alert] = result[:message]
-      @evaluator_invitations = @phase.evaluator_invitations
-      @existing_evaluators = @phase.evaluators
-      render :index
+      return
     end
+
+    handle_failed_invitation(result)
```

Comment:
I'm seeing some inconsistency with the error message display depending on the path it takes. Can we try to ensure that if there are any validation issues with the name(s) missing, that the `First & Last Name` label is always highlighted in red (as it does for new users)?

**existing user**
![Screenshot 2025-01-09 at 3 05 15 PM](https://github.com/user-attachments/assets/1af99793-e7c9-418a-aa41-9ae5bed1913a)

**new user**
![Screenshot 2025-01-09 at 3 05 38 PM](https://github.com/user-attachments/assets/82336d51-0812-4ec7-ac3c-73410a93b7a9)

Reply:
The first screenshot is also missing the inline error text label describing the issue.

Reply:
There is also a case where only the first name is entered, `First & Last Name` label should be red:
![Screenshot 2025-01-09 at 2 53 49 PM](https://github.com/user-attachments/assets/3a338b18-fd11-4846-890b-8b22f137cd65)